src/Dialect/ONNX/DialectBuilder.cpp

```diff
@@ -62,17 +72,26 @@ Value OnnxBuilder::matmul(Type Y, Value A, Value B, bool useGemm) const {
   return b.create<ONNXMatMulOp>(loc, Y, A, B);
 }
 
+Value OnnxBuilder::min(ValueRange inputs) const {
+  assert(inputs.size() >= 2 && "Expect at least two inputs");
```

Comment:
[nit]: could assert that all inputs have the same data type (using all_of or any_of).

Reply:
Done. Thanks!